### PR TITLE
Use USFIMR, HAND STACs

### PR DESCRIPTION
**Pull request by jamesmcclain**
See
   - https://github.com/azavea/raster-vision/pull/1004
   - https://github.com/azavea/raster-vision/pull/1007
   - https://github.com/azavea/raster-vision/pull/1008
   - https://github.com/azavea/raster-vision/pull/1016
   - https://github.com/azavea/raster-vision/pull/1017
   - https://github.com/azavea/raster-vision/pull/1018

This PR represents an intermediate result.  Work will continue when https://github.com/azavea/noaa-flood-mapping/issues/93 is available.  The image below is after one epoch of training (just to demonstrate that the null class is no longer being predicted and that water is being predicted).
test_preds

### CODE REVIEW

File: pipeline/patches/shm.diff [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+--- a/rastervision_pytorch_backend/rastervision/pytorch_backend/pytorch_semantic_segmentation_config.py
++++ b/rastervision_pytorch_backend/rastervision/pytorch_backend/pytorch_semantic_segmentation_config.py
+@@ -21,6 +21,7 @@ class PyTorchSemanticSegmentationConfig(PyTorchLearnerBackendConfig):
+             img_channels=pipeline.dataset.img_channels,
+             img_format=pipeline.img_format,
+             label_format=pipeline.label_format,
++            num_workers=self.num_workers,
+             augmentors=self.augmentors,
+             base_transform=self.base_transform,
+             aug_transform=self.aug_transform,
```
Comment (moradology) [inline pipeline/patches/shm.diff]: Yikes. This is pretty down and dirty. I assume this will get ironed out in a forthcoming release of RV

**Review by moradology — APPROVED** [review]
Looks good. The application of a diff inside a newly build docker image is likely to confuse down the road but makes sense for pushing things forward

File: pipeline/patches/shm.diff (hunk 1 above)
Comment (jamesmcclain) [reply pipeline/patches/shm.diff]: https://github.com/azavea/raster-vision/pull/1008/files

**Comment by jamesmcclain** [comment]
> Looks good. The application of a diff inside a newly build docker image is likely to confuse down the road but makes sense for pushing things forward

This will be removed in subsequent commits if/when these changes are accepted upstream and/or found not to be needed.

**Pull request merged**